GenderPayGap.BusinessLogic/Services/DraftFileBusinessLogic.cs

```diff
@@ -153,17 +151,17 @@ public async Task CommitDraftAsync(Draft draftToDiscard)
 
         #region private methods
 
-        private async Task<Draft> GetDraftOrCreateAsync(Draft resultingDraft, long userIdRequestingAccess)
+        private Draft GetDraftOrCreateAsync(Draft resultingDraft, long userIdRequestingAccess)

```

Comment:
It worth renaming any method with "Async" in its name

Reply:
Great spot! I'll change these